src/OpenLoco/Company.cpp

```diff
@@ -117,6 +119,19 @@ namespace OpenLoco
         }
     }
 
+    void Company::updateDaily()
+    {
+        // autopay loan
+        if (current_loan > 0 && cash > 0 && getGameStateExtensions().autopayLoan)
+        {
+            GameCommands::ChangeLoanArgs args{};
+            args.newLoan = current_loan - std::max<currency32_t>(0, std::min<currency32_t>(current_loan, cash.asInt64()));
+
+            GameCommands::setUpdatingCompanyId(id());
```

Comment:
Does this need to be set? I think it's auto set by the game command caller although could be wrong

Reply:
It isn't, without this the game just hard-crashes